01-ruby-fundamentals/exercises/problem-practice/problem.md

```diff
@@ -0,0 +1,18 @@
+# Math Problem Analyzer
```

Comment:
I really like this one! Aside from being good practice, having them work with our starter code is demonstrating good style.